airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/utils/docs.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+from typing import Any
+
+from markdown_it import MarkdownIt
+from markdown_it.tree import SyntaxTreeNode
+
+
+def remove_step_from_heading(heading: str) -> str:
+    if "Step 1: " in heading:
+        return heading.replace("Step 1: ", "")
+    if "Step 2: " in heading:
+        return heading.replace("Step 2: ", "")
+    return heading
```

Comment:
why do we stop after Step 2? I presume that many connectors will have steps beyond 2.

Reply:
thanks, really a good case with steps, as for example netsuite.md has more than 1 steps, we can try to strict compare step 1 and 2 as it is in connector standard template and skip others "Step" headers if they are after "Step 1:" header.

Reply:
see other message about whether we need to test this